From b0756d945065520072a2ec72c3ba4609f7738a78 Mon Sep 17 00:00:00 2001
From: Milan Crha <mcrha@redhat.com>
Date: Thu, 12 Feb 2015 13:46:45 +0100
Subject: [PATCH 0291/1800] Switch side of a wider border for signed/encrypted
 mail frame

A 4-pixel wide border looked fine for single emails, but once
the message was part of a conversation, with quoted replies,
the wider border was confusing, too close to the quotation line.
This change moves the wider border to the opposite side.
---
 mail/e-mail-display.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/mail/e-mail-display.c b/mail/e-mail-display.c
index cb6c27133e..798fd58b39 100644
--- a/mail/e-mail-display.c
+++ b/mail/e-mail-display.c
@@ -797,7 +797,7 @@ initialize_web_view_colors (EMailDisplay *display)
 	/* the rgba values below were copied from e-formatter-secure-button */
 	direction = gtk_widget_get_default_direction ();
 
-	if (direction == GTK_TEXT_DIR_LTR)
+	if (direction == GTK_TEXT_DIR_RTL)
 		style = "border-width: 1px 1px 1px 4px; border-style: solid; border-color: rgba(53%, 73%, 53%, 1.0)";
 	else
 		style = "border-width: 1px 4px 1px 1px; border-style: solid; border-color: rgba(53%, 73%, 53%, 1.0)";
@@ -807,7 +807,7 @@ initialize_web_view_colors (EMailDisplay *display)
 		".-e-mail-formatter-frame-security-good",
 		style);
 
-	if (direction == GTK_TEXT_DIR_LTR)
+	if (direction == GTK_TEXT_DIR_RTL)
 		style = "border-width: 1px 1px 1px 4px; border-style: solid; border-color: rgba(73%, 53%, 53%, 1.0)";
 	else
 		style = "border-width: 1px 4px 1px 1px; border-style: solid; border-color: rgba(73%, 53%, 53%, 1.0)";
@@ -817,7 +817,7 @@ initialize_web_view_colors (EMailDisplay *display)
 		".-e-mail-formatter-frame-security-bad",
 		style);
 
-	if (direction == GTK_TEXT_DIR_LTR)
+	if (direction == GTK_TEXT_DIR_RTL)
 		style = "border-width: 1px 1px 1px 4px; border-style: solid; border-color: rgba(91%, 82%, 13%, 1.0)";
 	else
 		style = "border-width: 1px 4px 1px 1px; border-style: solid; border-color: rgba(91%, 82%, 13%, 1.0)";
@@ -827,7 +827,7 @@ initialize_web_view_colors (EMailDisplay *display)
 		".-e-mail-formatter-frame-security-unknown",
 		style);
 
-	if (direction == GTK_TEXT_DIR_LTR)
+	if (direction == GTK_TEXT_DIR_RTL)
 		style = "border-width: 1px 1px 1px 4px; border-style: solid; border-color: rgba(91%, 82%, 13%, 1.0)";
 	else
 		style = "border-width: 1px 4px 1px 1px; border-style: solid; border-color: rgba(91%, 82%, 13%, 1.0)";
-- 
2.15.0

